python/graphframes/graphframe.py

```diff
@@ -16,7 +16,7 @@
 #
 
 from pyspark import SparkContext
-from pyspark.sql import Column, DataFrame, functions as sqlfunctions, SQLContext
+from pyspark.sql import Column, DataFrame, functions as sqlFuncs, SQLContext
```

Comment:
I actually think `sqlfunctions` make more sense? is there a reason for this change?